frontend/src/pages/item/_itemId/index.vue

```diff
@@ -246,94 +246,54 @@ import { getItemDetailsLink, getMediaStreams } from '@/utils/items';
 import { getItemizedSelect } from '@/utils/forms';
 import { useRemote } from '@/composables';
 
-export default defineComponent({
-  async setup() {
-    const { params } = useRoute();
-    const itemId = params.itemId;
-    const remote = useRemote();
-    const item = (
+const route = useRoute();
+const remote = useRemote();
+
+const item = ref<BaseItemDto>({});
+const currentSource = ref<MediaSourceInfo>({});
+const currentVideoTrack = ref<number>();
+const currentAudioTrack = ref<number>();
+const currentSubtitleTrack = ref<number>();
+
+const crew = computed((): BaseItemPerson[] =>
+  (item.value?.People ?? []).filter((person) =>
+    ['Director', 'Writer'].includes(person?.Type ?? '')
+  )
+);
```

Comment:
I left this here and in the other spots in this file and a few others because the return value reported by TypeScript and VS Code intellisense was the literal representation of the value (eg `{ Id: string | null | undefined, Name: string | null | undefined, ... }` instead of just `BaseItemPerson`). This just makes intellisense show the return value type as `BaseItemPerson`, so its purely for developer convenience.

Separately, I've updated all computed values that specify a return value to use the `computed<T>` form, I actually prefer that more too!

Reply:
@aweebs Fair enough, thanks!